components/Downloads/Release/VersionDropdown.tsx

```diff
@@ -21,19 +20,17 @@ const getDropDownStatus = (version: string, status: string) => {
 
 const VersionDropdown: FC = () => {
   const { releases, release, setVersion } = useContext(ReleaseContext);
-  const t = useTranslations();
 
   return (
     <Select
-      label={t('layouts.download.dropdown.version')}
       values={releases.map(({ status, versionWithPrefix }) => ({
         value: versionWithPrefix,
         label: getDropDownStatus(versionWithPrefix, status),
       }))}
       defaultValue={release.versionWithPrefix}
       onChange={setVersion}
       className="w-40"
-      inline={true}
```

Comment:
Why removing the ={true)? Usuall it's good to be explicit that this is a boolean.